examples/config/v0.4.1-selinux/go-uretprobe-counter/kustomization.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+# Patch the bytecode.yaml to change tag on the "url" field (which is an
+# image) to new value. This actually overwrites the image with the same value.
+patches:
+  - target:
+      kind: UprobeProgram
+      name: go-uprobe-counter-example
```

```suggestion
      name: go-uretprobe-counter-example
```
I think all the Program type are uretprobe, right?